### DISCUSSION TITLE
How to Style the scrollbar in the canvas

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#6042] [Q&A] [answered]
<img width="142" alt="Screenshot 2024-08-07 at 6 48 37 AM" src="https://github.com/user-attachments/assets/603d6a4c-700f-4143-b9fd-40db011c6b0d">

**Comment by artf** [comment, marked as answer]
Here the option: https://github.com/GrapesJS/grapesjs/blob/819bfcf44bb23d51990eba3b4330e9932df59cde/src/canvas/config/config.ts#L102-L107
